camply/config/notification_config.py

```diff
@@ -75,6 +75,18 @@ class PushbulletConfig:
         API_TOKEN = None
 
 
+class SlackConfig:
+    """
+    Slack Notification Config Class
+    """
+
+    try:
+        SLACK_WEBHOOK: str = environ["SLACK_WEBHOOK"]
+    except KeyError:
+        SLACK_WEBHOOK = None
+
+
+
```

Comment:
This looks to be the linters only actual complaint. One question though here @matthazinski, did you get a chance to get pre-commit running for black and isort? The code looks pretty impeccable so you might've. If not please check out some contributing details here: https://juftin.com/camply/contributing.html

Reply:
Thanks for the pointer, I missed that earlier. Was just using flake8 and missed this file.